workflows/execute_workflows.sh

```diff
@@ -150,6 +172,12 @@ CMD="$ROOT/quiver/summarize_benchmarks.py"
 $VENV $CMD
 echo "Done."
 
+# clean up
+rm -rf "$WORKSPACE_DIR"
+rm -rf "$ROOT"/work
+rm -rf "$WORKFLOW_DIR"/nf-results
+rm "$WORKFLOW_DIR"/results/*.json
+
```

Comment:
This is becoming a fairly complex script. The problem with long shell scripts is that it becomes quite hard to use just parts of it and testing it thoroughly. That's why I tend to use `Makefiles`, because it allows defining targets and dependencies, has builtin support for variable overrides. However, this can be optimized later.

I would strongly recommend `set -eu -o pipefail` to detect errors, undefined variables etc.

Reply:
See https://github.com/OCR-D/quiver-back-end/issues/44.